jsettlers.common/src/main/java/jsettlers/common/movable/EDirection.java

```diff
@@ -180,13 +180,17 @@ public static EDirection getDirectionOfMultipleSteps(int dx, int dy) {
 	}
 
 	public static EDirection getDirection(int dx, int dy) {
```

Comment:
This method gives a direction with the exact dx/dy. Some parts of the code depend on it. For what you want to do, use `getApproxDirection`.